addons/mail/static/src/core/common/thread_model.js

```diff
@@ -89,6 +89,12 @@ export class Thread extends Record {
         onDelete: (r) => r.delete(),
         sort: (m1, m2) => m1.id - m2.id,
     });
+    /**
+     * This is needed to be overrident in livechat
+     */
+    get channelMembersNoBot() {
+        return this.channelMembers;
+    }
```

Comment:
I'd rename to `membersThatCanSeen` or something similar.

Saying "no bot" refers to the actual implementation of the override, which doesn't make sense that this is not defined in `@mail`. And when we look at how this is used: to filter-out members for seen feature.
To add more problem with "NoBot", this is very confusing with OdooBot not being in the list, which is prone to making errors with future changes due to misleading field name.

Reply:
Alright!
I also updated the comment.